packages/basebridge/src/test-transactions.ts

```diff
@@ -0,0 +1,374 @@
+  ),
+  createTestCase(
+    ERC20_FROM_BASE_TO_ETHEREUM_TEST,
+    'bridge erc20 from base to ethereum, when amount not defined',
+    { amount: undefined },
+  ),
+]
+
+export const failingTestCases = [
+  // eth from ethereum
+  createTestCase(
+    ETH_FROM_ETHEREUM_TO_BASE_TEST,
+    'eth from ethereum to base, when sourceChainId is not correct',
+    {
+      sourceChainId: 5,
+    },
+  ),
+  createTestCase(
+    ETH_FROM_ETHEREUM_TO_BASE_TEST,
+    'eth from ethereum to base, when amount is not correct',
+    {
+      amount: '4000000',
+    },
+  ),
+  createTestCase(
+    ETH_FROM_ETHEREUM_TO_BASE_TEST,
+    'eth from ethereum to base, when tokenAddress is not correct',
+    {
+      tokenAddress: '0x990D07a69f23d69618e3Bb2444fA07f499193Fe4',
+    },
+  ),
+  createTestCase(
+    ETH_FROM_ETHEREUM_TO_BASE_TEST,
+    'eth from ethereum to base, when contractAddress is not correct',
+    {
+      contractAddress: '0x990D07a69f23d69618e3Bb2444fA07f499193Fe4',
+    },
+  ),
+  createTestCase(
+    ETH_FROM_ETHEREUM_TO_BASE_TEST,
+    'eth from ethereum to base, when contractAddress is not correct2',
+    {
+      contractAddress: ETHEREUM_ERC20_BRIDGE_ADDRESS,
+    },
+  ),
+  // erc20 from ethereum
+  createTestCase(
+    ERC20_FROM_ETHEREUM_TO_BASE_TEST,
+    'erc20 from ethereum to base, when sourceChainId is not correct',
+    {
+      sourceChainId: 5,
+    },
+  ),
+  createTestCase(
+    ERC20_FROM_ETHEREUM_TO_BASE_TEST,
+    'erc20 from ethereum to base, when amount is not correct',
+    {
+      amount: '4000000',
+    },
+  ),
+  createTestCase(
+    ERC20_FROM_ETHEREUM_TO_BASE_TEST,
+    'erc20 from ethereum to base, when tokenAddress is not correct',
+    {
+      tokenAddress: '0x990D07a69f23d69618e3Bb2444fA07f499193Fe4333',
+    },
+  ),
+  createTestCase(
+    ERC20_FROM_ETHEREUM_TO_BASE_TEST,
+    'erc20 from ethereum to base, when contractAddress is not correct',
+    {
+      contractAddress: '0x990D07a69f23d69618e3Bb2444fA07f499193Fe4',
+    },
+  ),
+  createTestCase(
+    ERC20_FROM_ETHEREUM_TO_BASE_TEST,
+    'erc20 from ethereum to base, when contractAddress is not correct2',
+    {
+      contractAddress: ETHEREUM_ETH_BRIDGE_ADDRESS,
+    },
+  ),
+
+  // eth from base
+  createTestCase(
+    ETH_FROM_BASE_TO_ETHEREUM_TEST,
+    'eth from base to ethereum, when sourceChainId is not correct',
+    {
+      sourceChainId: 5,
+    },
+  ),
+  createTestCase(
+    ETH_FROM_BASE_TO_ETHEREUM_TEST,
+    'eth from base to ethereum, when amount is not correct',
+    {
+      amount: '3333',
+    },
+  ),
+  createTestCase(
+    ETH_FROM_BASE_TO_ETHEREUM_TEST,
+    'eth from base to ethereum, when tokenAddress is not correct',
+    {
+      tokenAddress: '0x990D07a69f23d69618e3Bb2444fA07f499193Fe4333',
+    },
+  ),
+  createTestCase(
+    ETH_FROM_BASE_TO_ETHEREUM_TEST,
+    'eth from base to ethereum, when contractAddress is not correct',
+    {
+      contractAddress: '0x990D07a69f23d69618e3Bb2444fA07f499193Fe4333',
```

Comment:
Would remove this test. Since this will always match the ABI for an ETH deposit and not consider the token address.